src/pages/myNotes/notes.jsx

```diff
@@ -1,49 +1,121 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 
 import { ListItemText } from '@mui/material';
+import { ModalUnstyled } from '@mui/material';
 
 import ComponentInfo from './componentInfo';
 import NoteListItem from './noteListItem';
 import { todoList } from '../../constants/noteList';
-import { Container, NotesList, StyledList, StyledListComponent } from './styled';
+import { useLocalStorage } from '../../hooks/useLocaleStorage';
+import {
+    Backdrop,
+    ChangeDescription,
+    StyledInput,
+    Container,
+    EditButton,
+    ModalWindow,
+    NotesList,
+    StyledList,
+    StyledListComponent,
+} from './styled';
 
 const Notes = () => {
+    const [todosFromLocalStorage, setTodosFromLocalStorage] = useLocalStorage('todoList', todoList);
+
+    const [todos, setTodos] = useState([]);
     const [componentInfo, setComponentInfo] = useState({});
+    const [isEditModalOpen, setIsEditModalOpen] = useState(false);
+    const [changeDescriptionInputValue, setChangeDescriptionInputValue] = useState('');
+
+    useEffect(() => {
+        setTodos(todosFromLocalStorage);
+    }, []);
+
+    const setDescription = (element) => {
+        setChangeDescriptionInputValue(element.target.value);
+    };
+
+    const submitFormChanges = () => {
+        const changedTodoList = todos.map((todo) => {
+            if (todo.id === componentInfo.id) {
+                todo.description = changeDescriptionInputValue;
+            }
+            return todo;
```

Comment:
actually, it's almost the same, but try to avoid object mutation, you can replace it to
return todo.id === componentInfo.id?{...todo, description: changeDescriptionInputValue}: todo
_____________________________
and also you didn't fix my note about throwing as a parameter 'componentInfo'